### ISSUE TITLE
The command `sudo com.github.phase1geo.minder` works, while `com.github.phase1geo.minder` cannot work.

### ISSUE THREAD

**Issue by together00** [phase1geo/Minder#626]
Today, after I install Minder-1.16.4 (I compile it according to the tips), I find that I can launch it with the command `sudo com.github.phase1geo.minder`, while I cannot launch it with the command `com.github.phase1geo.minder`.

Interestingly, when I use the command `com.github.phase1geo.minder`, it outputs nothing, just like the figure below. Nothing happened, and I don't know what happened.

<img src="https://github.com/user-attachments/assets/ab7303e6-1745-4108-8fd9-7d98dc4f597b" width=50%>

My system is ubuntu 22.04. I have met this bug even with the Minder in the app center, whose version is 13.1... So, I am not sure what causes this bug, Minder, or my system itself.

Thanks for your kind help.

**Comment by together00** [comment]
This issue can be temporarily fixed by logging in/restarting again.

**Comment by phase1geo** [comment]
FYI, if you notice that this behavior is happening, try (in a terminal):  `ps -fe | grep minder`.  This will display the list of active processes running on your system which are related to Minder.  If there are one or more jobs with the name `./com.github.phase1geo.minder`, kill those processes using `kill -9 <process ID>` where the process ID is the first number in the process listing for the row (i.e., if I see something like `paralle+ 1773175 1773123 10 11:59 pts/0    00:00:00 ./com.github.phase1geo.minder` to kill the process I would say `kill -9 1773175`.

This is a "smaller hammer" way of dealing with the issue than restarting which is a big hammer approach.

**Comment by phase1geo** [comment]
Only one instance of Minder is allowed to run on your system at one time.  So if there is an existing Minder process and you attempt to start another one, it can cause a second process to run which is headless, meaning that if the existing Minder window is closed, it will end the process that originally started Minder, but the second process will remain running.  So when you attempt to start another instance of Minder, you see nothing (like your originally stated).

This is a quirk of Gtk, IMO, but probably one that can be dealt with (I just haven't been able to figure that part out yet).

**Comment by together00** [comment]
Thanks for your kind help, and the detailed response.

I will try to verify the effectiveness of this simple method when I meet the problem again.

**Comment by together00** [comment]
Yes, It works. I used the command `ps -fe | grep minder` you provided, and then kill the process.

<img src="https://github.com/user-attachments/assets/d3f40dd0-452f-4488-8780-fb017bbf0fec" width=80%>

Thanks for your kind help again. It really helps me a lot.

I am not sure whether this issue is common to others, yet it often bores me. Maybe it is because I used a laptop with ubuntu 24.04 installed (ubuntu 22.04, the time I first raise this issue).
 
Feel free to close this issue.
